packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -56,6 +56,7 @@ export interface IPushOptions {
     channelId?: string;
     categoryId?: string;
     mutableContent?: boolean;
+    fcmOptions?: { [key: string]: string };
     android?: { [key: string]: { [key: string]: string } };
     apns?: { payload: { aps: { [key: string]: { [key: string]: string } } } };
```

Comment:
In this interface you need to add to the `apns` property the `headers` type.
So far you can leave it like:
```
    apns?: { headers: { [key: string]: string }, payload: { aps: { [key: string]: { [key: string]: string } } } };
```

Reply:
Hi @p-fernandez 
I clean up the PR and added headers to apns